core/src/main/resources/hudson/model/Run/index.jelly

```diff
@@ -0,0 +1,73 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi, Matthew R. Harrah,
+Tom Huybrechts, id:cactusman, Romain Seguy
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:i="jelly:fmt">
+  <l:layout title="${it.fullDisplayName}">
+    <st:include page="sidepanel.jelly" />
+
+    <!-- no need for additional breadcrumb here as we're on an index page already including breadcrumb -->
+    <l:main-panel>
+      <div style="float:right; z-index: 1; position:relative; margin-left: 1em">
+        <l:hasPermission permission="${it.UPDATE}">
+          <st:include page="logKeep.jelly" />
+        </l:hasPermission>
+        <div style="margin-top:1em">
+          ${%startedAgo(it.timestampString)}
+          </div><div>
+          <j:if test="${it.building}">
+            ${%beingExecuted(it.executor.timestampString)}
+          </j:if>
+          <j:if test="${!it.building}">
+            ${%Took} <a href="${rootURL}/${it.parent.url}buildTimeTrend">${it.durationString}</a>
+          </j:if>
+          <j:if test="${!empty(app.nodes) or (it.builtOnStr != null and it.builtOnStr != '')}">
```

Comment:
This does not belong here since it references https://github.com/jenkinsci/jenkins/blob/a855a50c3196875d96858a65448db06e7ef9205f/core/src/main/java/hudson/model/AbstractBuild.java#L219-L226 for which there is no equivalent in `WorkflowRun` (or `ExternalRun`).